extra_stuff/lessons_in_progress/join_the_odin_community.md

```diff
@@ -0,0 +1,64 @@
+
+When you run into the desert of despair - where your code doesn't work or make sense to you anymore - you can find an oasis of knowledge and encouragement in our chat. We've all been there before!
+
+Remember that project you struggled with tirelessly and that you're so proud of finishing? You will get to share your work and progress with those that really appreciate how much hard work went into it.
+
+### Why chat is awesome for Odin:
+
+We are working hard to update existing lessons and produce new content so we would love to hear your feedback on how you are coping with the lessons and projects. We really want you to find the lessons fun, engaging and informative; while we would like you to find projects to be doable and challenging. So please tell us!
+
+## Oh foo bar!
+
+With great projects come great responsibility. When embarking on your projects, keep in mind that you do not need to know everything - BUT you do need to understand where you are going! Try to be as purposeful and specific as possible when describing the problem. Sometimes the problem can be a matter of your approach and not your coding ability: keep an open mind at all times.
+
+When the problem is a tough one, it's a good time to pause and take a breather. Break the problem down into little pieces and then decide what is really holding you back - we call this technique [rubber duck debugging](https://en.wikipedia.org/wiki/Rubber_duck_debugging). Who knows, you might have realised the solution already. If a solution still cannot be found, it's question time.
+
+### Asking your questions:
+
+A good way to phrase your question is to tell us the context:
+
+* What do you think the problem is?
+* What exactly do you want to happen?
+* What is actually happening? 
+* How did you get there?
+* What have you tried so far?
+
+### Formatting your questions:
+
+Making your questions in a readable format helps everyone debug them better; here are some ways to go about that:
+
+If you're having troubles with command-line stuff make sure you include *both your input and the error message* you're getting.
+
+In the chatroom, code can be displayed differently from normal sentences by using **backticks** - it's the button above your tab key - which are not to be confused with single quotation marks. If you wrap your code with backticks at **both** ends of the code you will get something like this: 
+
+`#one backtick at both ends`
+
+OR
+
+```
+#three backticks at the start
+Your Code
+#three backticks at the end
+```
+
+Show us a **screenshot** if you can't pinpoint the problem - you should use this especially to show the output of commands in the command line. (You will need to drag and drop your screenshot picture into the chat box to upload it on gitter chat).
+
+**Push your code to github!** If the puzzle still can't be solved, push your project to github so others can comb through and debug the code.
```

Comment:
If this lesson is coming after or before gearing up we will have to assume the student doesn't know what GitHub is yet. I think it would be a good idea to either remove this for now or add a note something like (You will learn how to do this in very soon)